projects/xtoken.js

```diff
@@ -1,7 +1,8 @@
+const Web3 = require('web3')
```

Comment:
why change this?

Reply:
While I am testing, I got this error:
```
Returned values aren't valid, did it run Out of Gas? You might also see this error if you are not using the correct ABI for the contract you are retrieving data from, requesting data from a block number that does not exist, or querying a node which is not fully synced.
``` 
And change config fixed that